stan/math/opencl/kernel_generator/load.hpp

```diff
@@ -133,6 +133,51 @@ class load_
    */
   inline matrix_cl_view view() const { return a_.view(); }
 
+  /**
+   * Sets view of the matrix depending on which part is written.
+   * @param top_diagonal Index of the top sub- or super- diagonal written with
+   * nonzero elements.
+   * @param bottom_diagonal Index of the top sub- or super- diagonal written
+   * with nonzero elements.
+   * @param top_zero_diagonal Index of the top sub- or super- diagonal written
+   * with zeros if it ie more extreme than \c top_diagonal. Otherwise it should
+   * be set to equal value as \c top_diagonal.
+   * @param bottom_zero_diagonal Index of the top sub- or super- diagonal
```

Comment:
Similar to above, order of arguments in the doxygen should match the one in the function signature.